app/metrics.yaml

```diff
@@ -122,7 +122,53 @@ events:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
-  preference_toggled:
+  whats_new_tapped:
```

Comment:
This ordering got all messed up. I pulled `preference_toggled` down into it's own section (per discussion with @sblatz and Marissa) and nested all onboarding clicks within that